superset-frontend/stylesheets/superset.less

```
@@ -582,3 +582,35 @@ hr {
   background-image: url('../images/icons/error_solid_small_red.svg') !important;
   background-position: -2px center !important;
 }
+
+// Ant-d overrides since these are injected as inline styles and can't
+// be overrided in emotion
```

```suggestion
// AntD overrides since these are injected as inline styles and can't
// be overridden in emotion
```
Teensy nits, but the real question I'm wondering is if these same styles couldn't be used in the main Menu component. These are all using !important, which should still work in that context, I would think. Then the styles are all consolidated with the component, and we're not growing the LESS codebase.